pkg/apis/serving/v1beta1/predictor_tfserving.go

```diff
@@ -32,6 +33,7 @@ var (
 	TensorflowServingGRPCPort            = "9000"
 	TensorflowServingRestPort            = "8080"
 	TensorflowServingGPUSuffix           = "-gpu"
+	TimeoutMilliSeconds                  int64
```

Comment:
This should be defined as a local variable in `GetContainer`

Reply:
addressed in 61de56e